diff --git a/drivers/i2c/busses/Makefile b/drivers/i2c/busses/Makefile
index 70e6205ae6f9..e44c0bfa3813 100644
--- a/drivers/i2c/busses/Makefile
+++ b/drivers/i2c/busses/Makefile
@@ -157,5 +157,6 @@ obj-$(CONFIG_I2C_XGENE_SLIMPRO) += i2c-xgene-slimpro.o
 obj-$(CONFIG_SCx200_ACB)	+= scx200_acb.o
 obj-$(CONFIG_I2C_FSI)		+= i2c-fsi.o
 obj-$(CONFIG_I2C_VIRTIO)	+= i2c-virtio.o
+obj-$(CONFIG_I2C_JBD4020)   += i2c-jbd4020.o
 
 ccflags-$(CONFIG_I2C_DEBUG_BUS) := -DDEBUG
